spec/actions/bump_version_update_changelog_create_pr_action_spec.rb

```diff
@@ -13,6 +13,29 @@
     let(:new_branch_name) { 'release/1.13.0' }
     let(:labels) { ['next_release'] }
 
+    it 'fails if version is invalid' do
+      allow(FastlaneCore::UI).to receive(:interactive?).and_return(true)
+      allow(FastlaneCore::UI).to receive(:confirm).with(anything).and_return(true)
+      allow(FastlaneCore::UI).to receive(:input).with('New version number: ').and_return('')
+      
+      expect(FastlaneCore::UI).to receive(:user_error!)
+        .with('Version number cannot be empty')
+        .once
+        .and_throw(:expected_error)
+
+      catch :expected_error do
```

Comment:
I've usually catched errors like this: https://github.com/RevenueCat/fastlane-plugin-revenuecat_internal/blob/806c2787a95522facb437be5cfad803b74f5e5fe/spec/helper/revenuecat_internal_helper_spec.rb#L100-L102 but this way will also catch the exact error message, so it's more complete 👍